cli/azd/cmd/i18n/en.yaml

```diff
@@ -0,0 +1,92 @@
+availableCommands: "Available Commands"
+awesomeAzdUrl: "https://azure.github.io/awesome-azd"
+azdHats: "https://aka.ms/azure-dev/hats"
+azdShortHelp: "is an open-source tool that accelerates the process of getting your application onto Azure and helps you manage it once it's there."
+azdUpNodeJsMongo: "azd up --template todo-nodejs-mongo"
+azdUpTemplate: "azd up --template"
+cmdCommonFooter: "Use {{.AzdRun}} to view examples and more information about a specific command."
+cmdConfigShort: "Manage azd configurations (ex: default Azure subscription, location)."
+cmdDeployShort: "Deploy the application's code to Azure."
+cmdDownShort: "Delete Azure resources for an application."
+cmdEnvShort: "Manage your environment settings."
+cmdGroupTitleAbout: "About, help and upgrade"
+cmdGroupTitleConfig: "Configure and develop your app"
+cmdGroupTitleManage: "Manage Azure resources and app deployments"
+cmdGroupTitleMonitor: "Monitor, test and release your app"
+cmdHelp: "Help for any command."
+cmdInfraShort: "Manage your Azure infrastructure."
+cmdInitShort: "Initialize a new application."
+cmdLoginShort: "Log in to Azure."
+cmdLogoutShort: "Log out of Azure."
+cmdMonitorShort: "Monitor a deployed application."
+cmdPipelineConfigShort: "Create and configure your deployment pipeline by using GitHub or Azdo Actions."
+cmdProvisionShort: "Provision the Azure resources for an application."
+cmdRestoreShort: "Restore application dependencies."
+cmdRootHelpFooterQuickStart: "Quick start: Deploying a sample application"
+cmdRootHelpFooterQuickStartDetail: "Initialize, provision, and deploy a template application by running the"
```

Comment:
Upon viewing, I think we can remove the extra indents on this paragraph and the ghost copy below. Like this:

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/6994218/221967961-1c4c9db0-3c3c-4131-9999-511731325fe6.png">